packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -2131,6 +2131,125 @@ account_pref:
     expires: never
     data_sensitivity:
       - interaction
+  bento_view:
+    type: event
+    description: |
+      User clicks on the bento menu on the top right of the account settings page
```

Comment:
Not sure we want "top right" here, on RTL it's flipped.

Reply:
Good catch, I'd copied directly from description in ticket. I'll update!